inclusive-design/wecount.inclusivedesign.ca
# fix: adjust styles, replace background image

## Description

Resolves #178, #179, #180, and #181.

## Steps to test

1. Review deploy preview.

**Expected behavior:** Confirm that the changes have been properly applied.

## Additional information

Not applicable.

## Related issues

- Resolves #178.
- Resolves #179.
- Resolves #180.
- Resolves #181.
